### DISCUSSION TITLE
Arm64 support for strimzi operator

### DISCUSSION THREAD

**Discussion by sudheesh-87** [strimzi/strimzi-kafka-operator#9713] [General]
### Suggestion / Problem

As discussed here - https://github.com/strimzi/strimzi-kafka-operator/issues/3357 , Multi-arch support for images are  considered as experimental , is it still experimental with new releases ? . there is no update on the latest release notes after 0.27.0 , we want to run strimzi in Arm64 in Production ..

### Documentation Link

https://github.com/strimzi/strimzi-kafka-operator/blob/main/CHANGELOG.md

**Comment by scholzj** [comment]
I do not think we consider Arm support experimental anymore. And I checked the docs and there does not seem to be anything suggesting that in the docs. The core of the Strimzi testing is done on amd64, but there are no known issues with Arm64.

**Reply by sudheesh-87** [reply]
Thanks @scholzj  for the confirmation ..
